src/js/utils/index.d.ts

```diff
@@ -34,6 +34,19 @@ declare const removeUndefined: <T extends object>(obj: T) => NonUndefinedProps<T
 
 export {isObject, deepFreeze, deepMerge, removeUndefined};
 
+/*
+ * Utility type for inferring the props type of a component.
+ *
+ * Example:
+ *
+ * ```typescript
```

Comment:
I usually include them since a lot of editors will pick up the formatting differently when you hover over values to see the documentation:

![image](https://user-images.githubusercontent.com/1895462/88440925-73a6d000-cddd-11ea-9af9-1462ef4e35df.png)

It doesn't seem to pick them up for the .d.ts files in this project for some reason, but I suppose it's worth keeping in case it gets moved around.

Reply:
ahhh very good point! didn't think of that